web/src/main/java/it/pagopa/pdv/person/web/controller/PersonController.java

```diff
@@ -43,15 +44,16 @@ public PersonController(PersonService personService) {
             notes = "${swagger.api.person.findById.notes}")
```

Comment:
Add a comment to describe the semantic for `id` param based on `namespace` param

Reply:
added with https://github.com/pagopa/pdv-ms-person/pull/42/commits/6e582c54c7f30a3eb32daf6ad94a088de35171ee